CardSharp/Card.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace CardSharp {
+    public static class Card {
+        public static Card<T> Make<T>(T dataModel) => Make(new[] { dataModel }).First();
+        public static Card<T>[] Make<T>(params T[] dataModels) => dataModels.Select(data => new Card<T>(data)).ToArray();
```

Comment:
I'd probably flip these so the single `Card<T> Make<T>` does the `new Card<T>(data)`, and then the array method calls into the first one.